crm_deduplicate_by_fuzzy_name/wizards/partner_merge.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp.osv import osv
+from openerp import api, fields, models, _
+from openerp.exceptions import Warning as UserError
+
+
+class BasePartnerMergeAutomaticWizard(models.TransientModel):
+    _inherit = "base.partner.merge.automatic.wizard"
+
+    # The field is named this way to not be automatically added as a "group by"
+    # option, which is performed in the code through the name of the field
+    fuzzy_name_group_by = fields.Boolean(
+        string='Similar name',
+        help="WARNING: This option is significantly slower than the others, "
+             "because the match cannot be performed in group.")
+    max_fuzzy_name_difference = fields.Integer(
+        string="Max. characters difference", default=2)
```

Comment:
I considered it, but as at the end, I have to "translate" it to characters difference, I see it less accurate. What is a 2% difference in a 5 letter word?